### PR TITLE
Browser telephone

### PR DESCRIPTION
**Pull request by punnie**

### CODE REVIEW

File: problems/f5_browser_phone.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Browser Telephone
+
+Ever wondered why we're in the year 2015 and you still can't use your browser to make a phone call to your grandmother while you browse Reddit for lolcats? Well, good news is that you actually can!
+
+## The Problem
+
+Browser telephony is at the core of our business, being we provide a browser based call centre and all. Turns out that the network piping and details — while not the simplest thing ever — are already handled by our provider, so all that's left to worry about is how to implement the most excellent browser phone ever known to Mankind.
```
Comment (ivoanjo) [inline problems/f5_browser_phone.md]: *being that we

File: problems/f5_browser_phone.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+It may be a browser extension, or something in a web page — basically very idea is fair game, as long as it is browser based, and doesn't need external plug-ins or applications.
```
Comment (ivoanjo) [inline problems/f5_browser_phone.md]: *every

**Comment by ivoanjo** [comment]
Simple, and completely to the core of our business. Awesome!

I just wanted to ask, what is the plan on the "*We'll set you up with an account and guide you around the API for the basics, along with common pitfalls we faced when building our product, but the rest is up to you!*" part?

File: problems/f5_browser_phone.md (hunk 1 above)
Comment (punnie) [inline problems/f5_browser_phone.md]: * meh.. granted
* granted
* do I have to? :cry:

**Comment by punnie** [comment]
I'm thinking:

— "Hey, here's a link to Twilio's API."
— "Cool" — said the grinning student, glancing in a mischievous way.
Moments pass uneventfully. Then, at the bring of dawn, torn by the countless hours head bashing through a dense mass of javascript code, the student was no longer grinning, but howling in pain:
— "Halp! I'm trying to close a connection and I'm failing miserably at it!" 
— "Ah, the famous `connection.close()` conundrum. Just use `.ignore()` to reset the client's perceived state over the connection, and everything will be alright.." — noted the Talkdesker, with a notoriously confident, yet humble look on their face, as if contemplating the horizon and the very fabric of reality itself.
The epiphany hit the student like a freight train. Suddenly everything made sense, and the world was back to normal.
— "Oh, it worked! It actually worked! You're great. Let me buy you beers. Forever!"
— "Appreciate it" — the Talkdesker's face glistening with the true light of knowledge — "but I've got a central to fix, my people need me!"
The Talkdesker's movement yield mathematical precision. Like clockwork, swift and gracious, he leaped to the air flying and leaving a trace of colourful light, leaving all students and passers by in a state of wonder. As he disappeared into the sky, words could distinctly be heard in such a way that made everybody calmer and more tranquil. Their problems just faded into pettiness as the following sounded over their heads:
— "Create a call centre in 5 minutes, just using your browser!"

**Comment by rfelix** [comment]
Much awesomeness, such entertaining!

**Comment by ivoanjo** [comment]
The PR is awesome :+1:

I would just add a link to the twilio js api: https://www.twilio.com/docs/client/twilio-js .

**Pull request merged**